lib/src/main/java/macrobase/analysis/summary/itemset/result/EncodedItemsetResult.java

```diff
@@ -0,0 +1,37 @@
+package macrobase.analysis.summary.itemset.result;
+
+import java.util.Set;
+
+public class EncodedItemsetResult {
+    private double support;
+    private double numRecords;
+    private double ratioToInliers;
+    private Set<Integer> items;
+
+    public EncodedItemsetResult(double support,
```

Comment:
Right not FPGrowth returns EncodedItemsetResult instead of ItemsetResult, because we don't want to pass the encoder in FPGrowth. The summarizer is in charge of decoding the results.